openshift/deploy/deploymentconfig/ciip-postgres-master.yml

```diff
@@ -68,6 +68,11 @@ objects:
                     secretKeyRef:
                       key: postgresqlAdminPassword
                       name: ${PREFIX}ciip-postgres-master
+                - name: GOOGLE_APPLICATION_CREDENTIALS
+                  valueFrom:
+                    secretKeyRef:
+                      key: credentials.json
+                      name: gcp-wksv3k-dev-cas-airflow-service-account-key
```

Comment:
Remember to update this name (checking the naming convention now)

Reply:
https://github.com/bcgov/cas-shelf/blob/03848c2415c8a3f1948b9496e9365ce116a84514/bcgov/main.tf#L60

This is where it's defined, so the secret name should be `gcp-${OC_PROJECT}-${PREFIX}ciip-postgres-service-account-key`